src/renderer/i18n/fr/settings.ts

```
@@ -62,6 +62,15 @@ const settings: SettingMessages = {
   'settings.expert.mayachain.title': 'URLs Mayachain',
   'settings.expert.midgardMaya.title': 'Midgard Mayachain',
   'settings.expert.mayanodeApi.title': 'API MayaNode',
+  'settings.expert.evm.title': 'URLs RPC des chaînes EVM',
+  'settings.expert.evm.eth.title': 'RPC Ethereum',
+  'settings.expert.evm.bsc.title': 'RPC BSC',
+  'settings.expert.evm.arb.title': 'RPC Arbitrum',
+  'settings.expert.evm.avax.title': 'RPC Avalanche',
+  'settings.expert.evm.base.title': 'RPC Base',
+  'settings.evm.rpc.error.url': 'URL RPC EVM invalide. Veuillez vérifier et réessayer',
+  'settings.evm.rpc.valid': 'URL RPC EVM valide',
+  'settings.evm.rpc.unhealthy': "Le point d'accès RPC ne répond pas. Vérifiez votre connexion ou essayez une autre URL.",
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)